conservation_hmm/conservation_hmm.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+from sys import argv
+from subprocess import DEVNULL, run
+
+
+def generate_MSA(FASTA_file, database_file, working_directory):
+    run(
+        "phmmer -o /dev/null -A {}{}.sto {} {}".format(
```

Comment:
The Python we are using is 3.7, so there is already support for f-strings. I would highly recommend using them.